### DISCUSSION TITLE
Config files on persistent volumes in Kubernetes deployment?

### DISCUSSION THREAD

**Discussion by svulpius-ns** [TwiN/gatus#625] [General]
Is it possible to configure gatus to read config files from a persistent volume in k8s? I know I can copy the whole chart and modify it to achieve this, but is it possible out of the box?

**Comment by TwiN** [comment]
Yes, you can just mount it to whatever path you want and then set `GATUS_CONFIG_PATH`, but why this over a config map?

**Reply by svulpius-ns** [reply]
Thank you! I'll be monitoring a large number of services and having configuration spread out over multiple config files will be easier for maintenance.
